modules/layers/src/text-layer/text-layer.js

```diff
@@ -64,6 +64,7 @@ const defaultProps = {
   sizeUnits: 'pixels',
   sizeMinPixels: 0,
   sizeMaxPixels: Number.MAX_SAFE_INTEGER,
+  backgroundColor: [0, 0, 0, 0],
```

Comment:
@xintongxia 
shouldn't this be 
`
backgroundColor: {type: 'color', value: [0, 0, 0, 0]},`